cli/exec/flags.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2021 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exec
+
+import (
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+var flags = []cli.Flag{
+	cli.BoolTFlag{
+		EnvVar: "WOODPECKER_LOCAL",
+		Name:   "local",
+		Usage:  "build from local directory",
+	},
+	cli.DurationFlag{
+		EnvVar: "WOODPECKER_TIMEOUT",
+		Name:   "timeout",
+		Usage:  "build timeout",
+		Value:  time.Hour,
+	},
+	cli.StringSliceFlag{
+		EnvVar: "WOODPECKER_VOLUMES",
+		Name:   "volumes",
+		Usage:  "build volumes",
+	},
+	cli.StringSliceFlag{
+		EnvVar: "WOODPECKER_NETWORKS",
+		Name:   "network",
+		Usage:  "external networks",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_DOCKER_PREFIX",
+		Name:   "prefix",
+		Value:  "drone",
+		Usage:  "prefix containers created by drone",
+		Hidden: true,
+	},
+	cli.StringSliceFlag{
+		Name:  "privileged",
+		Usage: "privileged plugins",
+		Value: &cli.StringSlice{
+			"plugins/docker",
+			"plugins/gcr",
+			"plugins/ecr",
+		},
+	},
+
+	//
+	// Please note the below flags are mirrored in the pipec and
+	// should be kept synchronized. Do not edit directly
+	// https://github.com/cncd/pipeline/pipec
```

Comment:
I think we removed `pipec`? Should we just remove that command?

Reply:
tbh no idear :)

Reply:
tbh no idear :)

Reply:
```suggestion
```